packages/adapter-cloudflare/index.js

```diff
@@ -28,13 +29,23 @@ export default function () {
 
 			const relativePath = posix.relative(tmp, builder.getServerDirectory());
 
-			writeFileSync(
-				`${tmp}/manifest.js`,
-				`export const manifest = ${builder.generateManifest({
-					relativePath
-				})};\n\nexport const prerendered = new Set(${JSON.stringify(builder.prerendered.paths)});\n`
+			builder.log.info(
+				`adapter-cloudfare is writing its own _headers file. If you have your own, you should duplicate the headers contained in: ${dest}/_headers`
```

Comment:
can we automate this, and copy the contents of `${builder.config.kit.files.assets}/_headers` into `${dest}/_headers` if it exists?

Reply:
We _could_, but I think that ought to be out-of-scope of this PR. We could optimistically concatenate the auto-generated headers to an existing `${dest}/_headers` file. However, I expect most users are running their build process and then copying their `src/_headers` to `${dest}/_headers` after the build completes. We still need to think about the best way to approach this scenario